code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -213,6 +213,10 @@
 	name = "Unholy Water"
 	id = "unholywater"
 	description = "Something that shouldn't exist on this plane of existence."
+	
+/datum/reagent/fuel/unholywater/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
```

Comment:
So the return doesn't go inside the if block... but I need to add a return inside the if block?